src/ol/layers/RoutingLayer.js

```diff
@@ -3,13 +3,17 @@ import { Vector as VectorSource } from 'ol/source';
 import { Vector } from 'ol/layer';
 import Layer from './Layer';
 
-const defaultStyleFunction = (feature) => {
+const defaultStyleFunction = (feature, resolution) => {
   const mot = feature.get('mot');
   const viaPointIdx = feature.get('viaPointIdx');
+  const minResolution = feature.get('minResolution');
+  const maxResolution = feature.get('maxResolution');
+  const inRange = resolution <= minResolution && resolution >= maxResolution;
```

Comment:
Can't we just return null in inRange is false ?  instead of setting strokes to null

Reply:
Good point, is done.